### DISCUSSION TITLE
Possible to not generate a project in typescript?

### DISCUSSION THREAD

**Discussion by lsbyerley** [blitz-js/blitz#1396] [General] [answered]
Is it possible to not generate a project in typescript and instead just regular javascript? I realize it's possible to use javascript files in the codebase but the generate command is very useful and provides a great jumpstart. It is, however, slow to get going for people who aren't typescript users. 

Just a thought and I'm sure you typescript truthers will disagree :)

**Comment by flybayer** [comment, marked as answer]
Hey! You can run `blitz new myapp --js`, but it's undocumented until https://github.com/blitz-js/blitz/issues/446 is fixed. This will definitely be fixed for 1.0

**Reply by lsbyerley** [reply]
Hey @flybayer thats awesome! Thanks for the reply, looking forward to the merge!

**Reply by lsbyerley** [reply]
@flybayer sorry one more question based on your response, so using the `--js` flag for `blitz generate` produces an error or isn't possible until the fix is released?

**Reply by flybayer** [reply]
Correct, it doesn't work right at the moment
